src/display/annotation_layer.js

```diff
@@ -1637,6 +1637,18 @@ class RadioButtonWidgetAnnotationElement extends WidgetAnnotationElement {
       storage.setValue(id, { value });
     }
 
+    if (value) {
+      // It's possible that multiple radio buttons are checked.
+      // So if this one is checked we just reset the other ones.
+      // (see bug 1864136).
```

Comment:
As mentioned in https://github.com/mozilla/pdf.js/pull/17318#issuecomment-1823450076, can you please extend this comment slightly?

Reply:
Sorry I thought it the patch comment.